invokeai/frontend/web/src/services/apiSlice.ts

```diff
@@ -42,7 +42,7 @@ const getModelId = ({ base_model, type, name }: ModelConfig) =>
   `${base_model}/${type}/${name}`;
 
 export const api = createApi({
-  baseQuery: fetchBaseQuery({ baseUrl: 'http://localhost:5173/api/v1/' }),
+  baseQuery: fetchBaseQuery({ baseUrl: 'http://localhost:9090/api/v1/' }),
```

Comment:
Can this utilize the same OPENAPI.BASE that is used elsewhere? Maybe by only defining a relative URL?

Reply:
Yeah. Just updated this for the time being.

When we are shipping 3.0, we'll need to retrieve the host and port from the backend (possibly via a new route) and use that data to set the proxy and the baseURL I think.